### ISSUE TITLE
Layers position mismatch 

### ISSUE THREAD

**Issue by novikov-alexander** [qgis/qwc2-demo-app#122]
After  pulling commit [Fix WMS tilegrid resolutions not matching view resolutions] https://github.com/qgis/qwc2/commit/f823304844b203ca5478f328eb24984b5c3b1b04) I've found out that  size and position of my vector layers don't match my background layer in all parts of map except central image.

![regions](https://user-images.githubusercontent.com/8073853/48058990-06967580-e1c9-11e8-8cb1-34bd5d3d38bb.PNG)
Green regions are regions of Russian Federation but as you see they are placed on other countries on this map.

I tried to debug WMSLayer.js and discovered that if I pass undefined instead TileGrid in constructor than everything works.

In what problem may be?

**Comment by manisandro** [comment]
Can you try setting the tile grid origin? See

https://openlayers.org/en/latest/apidoc/module-ol_tilegrid_TileGrid-TileGrid.html

**Comment by novikov-alexander** [comment]
@manisandro I can't understand what you exactly mean.
Origin is already set by constructor.

![debugger](https://user-images.githubusercontent.com/8073853/48060947-8246f100-e1ce-11e8-8a55-50bc3e09e0d7.png)

I discovered that the problem is not in my project, but it is currently in master and it is exactly in f823304 commit.

I've just opened current master of qwc2-demo-app, QWC2 Test theme, and rotated the Earth one time clockwise. 
That is what I've seen.

![zurich](https://user-images.githubusercontent.com/8073853/48061090-e23d9780-e1ce-11e8-89f7-fec4b6f87913.PNG)
